samples/javascript/README.md

```diff
@@ -0,0 +1,37 @@
+# Export API Javascript Example
+
+In this JavaScript example, build durations are calculated and written out to the JavaScript console.
+
+## Setup
+
+To run this sample:
```

Comment:
I would change this with `Setup` and add a step for the checkout of the repo. Without it, the sentence `within this directory` lacks context.